app/models/user.rb

```diff
@@ -14,3 +18,9 @@ def most_recent_posts
     posts.order(created_at: :desc).limit(3)
   end
 end
+
+private
+
+def make_admin
+  update(role: 'admin')
+end
```

Comment:
- [ ] Since you are using the `admin?` method in the `app/models/ability.rb` file to check if **_a user is an admin_**, you can create the `admin?` method this way:

```ruby
def admin?
  role == 'admin'
end
``` 
This will set up the abilities of a user properly instead of the current method that you have. Kindly erase the `make_admin?` method. 😎 🖱️ 

#### N.B: Please, don't make the `admin?` method private. 👍